scripts/create_json/create_json.py

```diff
@@ -1,14 +1,34 @@
 import os
 import sys
+
 import pandas as pd
 
 data_path = os.getcwd()
 df = pd.read_excel(
-    os.path.join(data_path, "Q&As.xlsx", ),
-    sheet_name="Check lists")
+    os.path.join(
+        data_path,
+        "Q&As.xlsx",
+    ),
+    sheet_name="Checklists FINAL",
+)
 
-df = df[['PS', 'Checkpoint', 'Question', 'Support Notes', 'Project areas']]
-df = df.rename(columns={'PS': 'Barrier', 'Question': 'Text', 'Support Notes': 'SupportNotes', 'Project areas': 'Organization'})
+df = df[
+    [
+        "PS",
+        "Checkpoint (pink = new text)",
+        "Question",
+        "Support Notes",
+        "Responsible/ Related to: (eller noe i den duren)",
+    ]
+]
+df = df.rename(
+    columns={
+        "PS": "Barrier",
+        "Question": "Text",
+        "Support Notes": "SupportNotes",
+        "Responsible/ Related to: (eller noe i den duren)": "Organization",
+    }
+)
```

Comment:
If we rename first, can we get a prettier pick of columns? 

```suggestion
df = df.rename(
    columns={
        "PS": "Barrier",
        "Question": "Text",
        "Support Notes": "SupportNotes",
        "Responsible/ Related to: (eller noe i den duren)": "Organization",
    }
)
df = df[
    [
        "Barrier",
        "Text",
        "SupportNotes",
        "Organization",
    ]
]
```

Reply:
Agreed and done =)